packages/e2e-tests/specs/editor/various/font-size-picker.test.js

```diff
@@ -114,6 +120,10 @@ describe( 'Font Size Picker', () => {
 		await pressKeyTimes( 'Backspace', 5 );
 		await page.keyboard.press( 'Enter' );
 
+		// Disable reason: Wait for changes to apply.
+		// eslint-disable-next-line no-restricted-syntax
+		await page.waitForTimeout( 1000 );
```

Comment:
1 second to wait is a lot. It should be further investigated.

Reply:
I think this doesn't need to be 1 whole second @ItsJonQ ?